schemas/Account.json

```diff
@@ -31,6 +31,10 @@
     "is_admin": {
       "description": "admin flag",
       "type": "boolean"
+    },
+    "disabled": {
+      "description": "Account is disabled",
```

Comment:
It is reversible. The idea is to allow admins to disable or re-enable accounts at any time. I will update the description.